src/components/RoutingSlip/CreateRoutingSlipCashPayment.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <v-form ref="createRoutingSlipCashPaymentForm">
+    <v-form ref="createRoutingSlipCashPaymentForm" :disabled="isViewMode">
```

Comment:
is it better to use v-can like sbc auth?

Reply:
since there is only page level access, we can ignore for now?